resources/views/hr/job/create.blade.php

```diff
@@ -42,6 +42,14 @@
                             <option value="internship" {{ old('type') == 'internship' ? 'selected' : '' }}>Internship</option>
                         </select>
                     </div>
+                    <div class="col-md-3 form-group">
+                        <label for="domain" class="fz-14 leading-none text-secondary mb-1">Domain<label class="text-danger">*</label></label>
```

Comment:
Let's not use label inside label. Please use `<span>` or `<strong>`
```suggestion
                        <label for="domain" class="fz-14 leading-none text-secondary mb-1">Domain<strong class="text-danger">*</strong></label>
```

Please update for edit page as well.